docs/porting/nuget-packages.md

```diff
@@ -1,5 +1,36 @@
-# 🔧 Porting NuGet packages
+# Porting NuGet Packages
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
+This article covers how to get your ported .NET Framework code to NuGet.  Although the packaging format has not changed, your code has not fundamentally changed with .NET Core code (compared with .NET Framework code), there are different avenues with which you can package your code.
```

Comment:
This introduction is a bit confusing and did you mean to say that the code has fundamentally changed? 
Maybe something like this:
This article covers how to generate NuGet packages for the code you ported from the .NET Framework to .NET Core. Although the packaging format has not changed, your code has fundamentally changed with .NET Core (when compared to the original .NET Framework code), so there are different options you can consider when you're packing your code.